release.config.js

```diff
@@ -3,6 +3,12 @@ module.exports = {
   plugins: [
     ["@semantic-release/commit-analyzer", { preset: "conventionalcommits" }],
     ["@semantic-release/release-notes-generator", { preset: "conventionalcommits" }],
+    [
+      "@semantic-release/exec",
+      {
+        prepareCmd: "yarn build",
```

Comment:
This is nice because now when we run `yarn semantic-release` it'll ensure the app is built before releasing.